app/bundles/ReportBundle/Resources/views/Graph/Table.html.twig

```diff
@@ -21,25 +21,27 @@
                     </thead>
                     <tbody>
                         {% for rowKey, row in graph.data %}
-                        <tr>
-                            {% for cellName, cell in row %}
-                                {% if graph.data[0].id is defined and 'title' == cellName and graph.link is defined %}
-                                    <td>
-                                        <a href="{{ path(graph.link, {'objectAction': 'view', 'objectId': row.id}) }}" data-toggle="ajax">
-                                            {{ cell }}
-                                        </a>
-                                    </td>
-                                {% elseif 'id' != cellName %}
-                                    <td>
-                                        {% if cell is numeric %}
-                                            {{ cell }}
-                                        {% else %}
-                                            {{ assetMakeLinks(cell) }}
+                            {% if row['id'] is not null %}
```

Comment:
This caused a bug: https://github.com/mautic/mautic/issues/13695

Can you please make a fix? I think you have to check if the ID key defined first like on line 27

Reply:
ok, I will.